### PR TITLE
Release standalone geofire-utils library

### PR DESCRIPTION
**Pull request by samtstern**
See #197, which was accidentally merged too fast.

@puf when merging this one please use "Squash and Merge" we don't want all 15 commits to end up on `master`

### CODE REVIEW

File: packages/geofire-common/rollup.config.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { uglify } from 'rollup-plugin-uglify';
 import pkg from './package.json';
 
 const GLOBAL_NAME = 'geofire';
+const UTILS_NAME = 'geofire-utils';
```
Comment (samtstern) [inline packages/geofire-common/rollup.config.js:8]: I don't think we're using this anywhere?

File: packages/geofire-common/test/Main.test.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import * as chai from 'chai';
+
+const expect = chai.expect;
+
+describe('GeoFire GeoCallbackRegistration Tests:', () => {
```
Comment (samtstern) [inline packages/geofire-common/test/Main.test.ts:5]: I think this test can probably be removed (or improved) not sure how it got here.

File: packages/geofire/src/index.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
 export { GeoCallbackRegistration } from './GeoCallbackRegistration';
 export { GeoFire } from './GeoFire';
 export { GeoQuery } from './GeoQuery';
-export { GeoFireTypes } from './GeoFireTypes';
\ No newline at end of file
+export * from './GeoFireTypes';
```
Comment (samtstern) [inline packages/geofire/src/index.ts:4]: Is there a reason for this change?  Or was this part of the rollup phase?

File: packages/geofire-common/test/Main.test.ts (hunk 2 above)
Comment (puf) [reply packages/geofire-common/test/Main.test.ts:5]: It's the only test in there, so probably a placeholder? I've commented it out, to at least leave it as an example of what tests should look like.

File: packages/geofire/src/index.ts (hunk 3 above)
Comment (puf) [reply packages/geofire/src/index.ts:4]: The [GeoFireTypes.ts](https://github.com/firebase/geofire-js/pull/203/files#diff-0a9e4aa8acec255b72e32ce315d46310ac157cb6f842bbf1600859abae01b5d5) is new, meant to isolate types that `geofire-utils` needs from types that only `geofire` proper need. In this last commit I removed the unnecessary namespacing after a chat with @jamesdaniels on this antipattern.

**Pull request merged**